frame/evm/src/lib.rs

```diff
@@ -145,6 +145,11 @@ decl_event! {
 		Log(Log),
 		/// A contract has been created at given address.
 		Created(H160),
+
+		/// A batch of all indices written during last call (see "storage_tracing" feature)
+		StorageWritten(backend::StorageWritten),
```

Comment:
Same as those two items. They should be hidden behind the `storage-tracing` feature.